build/zip.py

```diff
@@ -10,6 +10,27 @@
   'libnode.so'
 ]
 
+EXTENSIONS_TO_SKIP = [
+  '.pdb'
+]
+
+PATHS_TO_SKIP = [
+  'angledata',
+  'swiftshader',
+  'resources/inspector'
+]
+
+def skip_path(dep):
+  should_skip = False
+  for path in PATHS_TO_SKIP:
+    if dep.startswith(path):
+      print 'Skipping: '+dep
+      should_skip = True
+  for extension in EXTENSIONS_TO_SKIP:
+    if dep.endswith(extension):
+      print 'Skipping: '+dep
+      should_skip = True
```

Comment:
I think you forgot to `return should_skip`! Also, this can be written a bit simpler:
```py
should_skip = (
    any(dep.startswith(path) for path in PATHS_TO_SKIP) or
    any(dep.endswith(ext) for ext in EXTENSIONS_TO_SKIP))
if should_skip:
  print("Skipping {}".format(dep))
return should_skip
```

Reply:
oops... I refactored that and forgot that part.  I'll clean it up.